ubuntu/ubuntu-22.x/ubuntu-22.04-hpc/install_rccl.sh

```diff
@@ -1,6 +1,20 @@
 #!/bin/bash
 set -ex
 
+#update CMAKE
+pushd /tmp
+CMAKE_VER="3.30.6"
```

Comment:
As I called out in a PR a couple of weeks ago, I strongly suggest using the common utility functions to retrieve the version from versions.json, verify your download, and write the component version used to the component_versions.txt so that it's stored in the image for reference.

Most files have examples of doing this, but https://github.com/Azure/azhpc-images/blob/master/common/install_azcopy.sh is fairly simple. This should be done for all components that are being installed.

Reply:
Trying to use the `download_and_verify.sh` file for cmake and rocm but this file does not take the path to store the downloaded file and calling something like this breaks it:
```
pushd /tmp
download_and_verify.sh <url> <sha256>
```
as it stands, I think I have to download the package to the source tree and remove it when it is done.

Reply:
usually we download into the working directory and then delete